README.md

```diff
@@ -65,7 +65,7 @@ Copy `.env.dist` to `env.local` and update the relevant values, e.g., a UI deplo
 
 The following files can/should be replaced before running the build.
 
-* `cp app-id.dist.json public/app-id.json` and update the ids appropriately for U2F usage
+* `cp app-id.dist.json public/app-id.json` and update the ids appropriately for WebAuthn usage
```

Comment:
I think this is actually specific to U2F. @fillup, do we still need the `app-id.json` file for U2F backwards compatibility?

Reply:
I do not believe we need the file to exist anymore. We still need to use the URL as part of the login process, but I don't believe the browser fetches the file anymore.

Reply:
Oh true, unless I'm misremembering, I don't think it's needed any longer unless we want to support U2F still. Given this is profile ui and only supports register, I don't think this is needed.